### DISCUSSION TITLE
backdropbuild.com

### DISCUSSION THREAD

**Discussion by stsp** [dosemu2/dosemu2#2258] [General]
Have anyone heard of this project:
https://backdropbuild.com/faq
They sent me an invitation, and well,
I suppose they sent the similar invitation
to thousands of github project maintainers,
as dosemu2 is by very far not the most
active on github...
They offer a hackathon on your own project,
and the money prizes for success... Sounds
like gsoc initially, but the more I read their
site, the less I understand what is this all about. :)
Maybe someone here have heard of them?

**Comment by jschwartzenberg** [comment]
I have not heard from them. I guess they did look at bigger/famous projects. No idea is this would make sense for DOSEMU2.
